edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/DeviceMessageHandlerTest.cs

```diff
@@ -323,6 +323,7 @@ public async Task MessageCompletionMismatchedResponseTest()
 
             await deviceMessageHandler.ProcessMessageFeedbackAsync(Guid.NewGuid().ToString(), FeedbackStatus.Complete);
 
+            await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
Does this test fail without this? Or is it flaky without this?

Reply:
The test is flaky without it.